.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -0,0 +1,63 @@
+name: Bug report
+description: Submit a bug report
+labels: ["type-bug"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+          # New to Python?
+
+          The issue tracker isn't the right place to get help.
+          Consider instead:
+
+            - reading the [Python tutorial](https://docs.python.org/3/tutorial/)
+            - posting on [Discourse](https://discuss.python.org/c/users/7)
+            - emailing [python-list](https://mail.python.org/mailman/listinfo/python-list)
```

Comment:
This could also be collapsed in a sentence (instead of a list), since the form is about reporting issue, and not about getting help (even though I understand that you want this warning to be noticeable).

Reply:
I see your point, but I think I prefer bullets here. These are distinct options, after all, and I think it makes it easier/more appealing for people to read.

Reply:
Fair enough, the suggestion came from the fact that I wanted to make the disclaimer less prominent, but if we want to keep it more visible a list of bullets is indeed better.